sites/hashdev/src/_pages/blog/5_build-blocks-in-any-language.mdx

```diff
@@ -10,7 +10,7 @@ date: "2022-06-14"
 
 One of the main motivations behind recent Block Protocol updates has been to enable blocks to be written in a wider variety of languages. To make that possible, the ways in which blocks and embedding applications interact have changed significantly. As of version 0.2 of the specification, Block Protocol requests and responses are handled through DOM `CustomEvents` defined in accordance with JSON Schemas.
 
-It's now a lot easier to create blocks that are not tied to JavaScript, TypeScript, or React. As long as a block can use browser `Event`s, it can implement the Block Protocol. To show what the changes imply for adventurous block authors, this post will be a technical walk-through of how the Block Protocol can be implemented and used in a language that is not TypeScript. Specifically, we're going to build a block in F#—but the methodology and implementation details are applicable for other languages that support JavaScript as a transpilation target and JavaScript interoperability.
+It's now a lot easier to create blocks that are not tied to JavaScript, TypeScript, or As long as a block can use browser `Event`s, it can implement the Block Protocol. To show what the changes imply for adventurous block authors, this post will be a technical walk-through of how the Block Protocol can be implemented and used in a language that is not TypeScript. Specifically, we're going to build a block in F#—but the methodology and implementation details are applicable for other languages that support JavaScript as a transpilation target and JavaScript interoperability.
```

Comment:
This this is unintentional?

Reply:
Hmm it's strange that this has shown up in a diff for you. I fixed this on Friday in https://github.com/hashintel/hash/pull/847/commits/6a6bd2328c9e6240d6c9be0162251adca2a3da5a. Can it be that you've been looking at the old version of this PR somehow?

Reply:
Gotcha, yeah not sure how that happened